src/main/java/ch/epfl/biop/operetta/commands/OperettaImporter.java

```diff
@@ -53,27 +53,46 @@ public class OperettaImporter implements Command {
     @Parameter
     CommandService cs;
 
+    private enum XMLFILE {
+        V5("Index.idx.xml", "PerkinElmer Harmony V5"),
+        V6("Index.xml", "PerkinElmer Harmony V6");
+
+        private final String description;
+        private final String indexFileName;
+
+        XMLFILE(String indexFileName, String description) {
+            this.indexFileName = indexFileName;
+            this.description = description;
+        }
+        private String getIndexFileName() { return this.indexFileName; }
+        public String getDescription() { return this.description; }
+    }
+
     @Override
     public void run() {
+        File f = null;
```

Comment:
Change the way candidates are working here to account for the memo file part

Re-add memoization check part